bindings/python/pyproject.toml

```diff
@@ -41,7 +41,7 @@ benchmark = [
   "boto3-stubs[essential]",
 ]
 docs = ["pdoc"]
-test = ["behave", "pytest", "python-dotenv"]
+test = ["behave", "pytest", "python-dotenv", "pytest-asyncio"]
```

Comment:
```suggestion
test = ["pytest", "python-dotenv", "pytest-asyncio"]
```

`behave` is unused now?

Reply:
Have been removed in https://github.com/apache/incubator-opendal/pull/3253

Reply:
Yes, I removed it in #3253

Reply:
Right, just need to resolve conflicts.

Reply:
> Yes, I removed it in #3253

Oh, I got @messense's point. This PR needs a rebase or merge.